.github/workflows/main-tier0.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: konveyor/tackle2-operator/.github/actions/install-tackle@main
       - uses: actions/setup-go@v3
         with:
-          go-version: '1.19'
+          go-version: '1.22'
```

Comment:
No big deal, but I'd prefer use the same go versions for on-PR and nightly-executed workflows.

Reply:
Yeah this was a change that had to be reverted sorry about leaving that in